index.html

```diff
@@ -109,87 +133,76 @@ <h2>Tonic</h2>
-                        <li class="pull">
-                            <img class="logos" src="images/Ellipse01.svg">
-                            <span class="wording">javascript</span>
-                        </li>
-                        <li class="pull">
-                            <img class="logos" src="images/Ellipse02.svg">
-                            <span class="wording">html</span>
-                        </li>
-                        <li class="pull">
-                            <img class="logos" src="images/Ellipse03.svg">
-                            <span class="wording">css</span>
-                        </li>
-                    </ul>
-                    <div class="languages">
-                        <h3>Frameworks</h3>
-                        <span><i class="fa fa-chevron-right"></i></span>
-                    </div>
-                    <div class="languages">
-                        <h3>Skills</h3>
-                        <span><i class="fa fa-chevron-right"></i></span>
-                    </div>
+                        <div class="languages">
+                            <h3>Languaje</h3>
+                            <span><i class="fa fa-chevron-down "></i></span>
+                        </div>
+                        <ul>
+                            <li class="pull">
+                                <img class="logos" src="images/Ellipse01.svg">
+                                <span class="wording">javascript</span>
+                            </li>
+                            <li class="pull">
+                                <img class="logos" src="images/Ellipse02.svg">
+                                <span class="wording">html</span>
+                            </li>
+                            <li class="pull">
+                                <img class="logos" src="images/Ellipse03.svg">
+                                <span class="wording">css</span>
+                            </li>
+                        </ul>
+                        <div class="languages">
+                            <h3>Frameworks</h3>
+                            <span><i class="fa fa-chevron-right"></i></span>
+                        </div>
+                        <div class="languages">
+                            <h3>Skills</h3>
+                            <span><i class="fa fa-chevron-right"></i></span>
+                        </div>
                 </div>
-   </section>
+        </section>
+    </main>
+    <footer>
+        <div class="bottom">
+            <h2 class="white">Contact me </h2>
+            <p class="#">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/jlberbessi@gmail.com"
+            method="POST">
     
+                <input type="text" id="fname" name="fname" placeholder="Your Name" value="John" maxlength="30"><br>
```

Comment:
- [ ] Kindly, add a `required` attribute to prevent the user from submitting empty input 💯

![image](https://user-images.githubusercontent.com/45231276/193523062-eab0a8b3-599f-4f45-acc9-ac6d1a95fcb2.png)

Reply:
Done. Thanks a lot for the suggestion